client/my-sites/patterns/components/get-access-modal/index.tsx

```diff
@@ -25,6 +26,7 @@ export const PatternsGetAccessModal = ( {
 	tracksEventHandler,
 }: PatternsGetAccessModalProps ) => {
 	const locale = useLocale();
+	const { hasTranslation } = useI18n();
```

Comment:
Can we use `useHasEnTranslation` from `@automattic/i18n-utils` here? It has the added benefit of checking whether the active locale is English (including its variations)